packages/loader/driver-utils/src/messageRecognition.ts

```diff
@@ -0,0 +1,22 @@
+/*!
```

Comment:
I'm not sure why these definitions need to be in driver utils - this seems a bit wrong place for them.
container-definitions seems like a bit better place. This will make it not possible for drivers to leverage such definitions, but I think it's fine - the only place today where drivers need it is ODSP snapshot fetching logic, for telemetry, and I'd rather have an explicit check there for type == "op" and not worry much about it.

Reply:
1. definition shouldn't contain code
2. odsp-driver uses it.  So it needs to be in the driver layer